### PR TITLE
feat: add first time configs

### LINKED ISSUE #96: Add first time contributor message
# Specification

The two business problems to solve are: 

1. a custom message that partners can use to greet new hunters
2. show a list of all of the bot commands that the new hunter can invoke.

We must create new config properties: 
- [ ] `FIRST_TIME_CONTRIBUTOR_GREETING_HEADER`: string;
- [ ] `FIRST_TIME_CONTRIBUTOR_GREETING_HELP_MENU`: boolean;
- [ ] `FIRST_TIME_CONTRIBUTOR_GREETING_FOOTER`: string;

Post the header first, then the help menu, and lastly the footer in a single "first time contributor greeting" comment.

- [ ] Be sure to also update the README.md with these config properties.

# Example Input

```
FIRST_TIME_CONTRIBUTOR_GREETING_HEADER="Thank you for contributing to UbiquiBot! Please be sure to set your wallet address before completing your first bounty so that the automatic payout upon task completion will work for you."
FIRST_TIME_CONTRIBUTOR_GREETING_HELP_MENU=true
FIRST_TIME_CONTRIBUTOR_GREETING_FOOTER="###### Also please star this repository and [@ubiquity/devpool-directory](https://github.com/ubiquity/devpool-directory/) to show your support. It helps a lot!"
```

# Example Output

Thank you for contributing to UbiquiBot! Please be sure to set your wallet address before completing your first bounty so that the automatic payout upon task completion will work for you. 

```
- /assign: Assign the origin sender to the issue automatically.
- /unassign: Unassign the origin sender from the issue automatically.
- /help: List all available commands.
- /payout: Disable automatic payment for the issue.
- /multiplier: Set bounty multiplier (for treasury)
- /allow: Set access control. (Admin Only)
- /wallet: <WALLET_ADDRESS | ENS_NAME>: Register the hunter's wallet address. 
  ex1: /wallet 0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266 
  ex2: /wallet vitalik.eth
```

### PR DESCRIPTION
**Pull request by EtherealGlow**
Resolves #96

### CODE REVIEW

**Comment by EtherealGlow** [comment]
this can only be merged when [this](https://github.com/ubiquity/ubiquibot/issues/711) is solved

File: src/handlers/comment/handlers/first.ts [hunk 1]
```diff
@@ -7,30 +7,37 @@ export const verifyFirstCheck = async (): Promise<void> => {
...
+    const response = await context.octokit.rest.search.issuesAndPullRequests({
       q: `is:issue repo:${payload.repository.owner.login}/${payload.repository.name} commenter:${payload.sender.login}`,
       per_page: 2,
     });
-    const response_pr = await context.octokit.rest.search.issuesAndPullRequests({
-      q: `is:pull-request repo:${payload.repository.owner.login}/${payload.repository.name} commenter:${payload.sender.login}`,
-      per_page: 2,
-    });
-    if (response_issue.data.total_count + response_pr.data.total_count === 1) {
+    if (response.data.total_count === 1) {
```
Comment (whilefoo) [inline src/handlers/comment/handlers/first.ts:21]: it looks like you have some unintended changes here?

**Review by whilefoo — CHANGES_REQUESTED** [review]

**Review by 0xcodercrane — APPROVED** [review]
waiting for #711 merged

**Comment by EtherealGlow** [comment]
@0xcodercrane #711 is merged

**Comment by whilefoo** [comment]
@EtherealGlow resolve the conflicts

**Review by 0xcodercrane — APPROVED** [review]

File: src/handlers/comment/handlers/first.ts [hunk 2]
```diff
@@ -28,9 +31,17 @@ export const verifyFirstCheck = async (): Promise<void> => {
         per_page: 100,
       });
       const isFirstComment = resp.data.filter((item) => item.user?.login === payload.sender.login).length === 1;
-      if (isFirstComment) {
+      if (isFirstComment && enabled) {
```
Comment (molecula451) [inline src/handlers/comment/handlers/first.ts:34]: this var could've been refactored to something else, "enabled" sounds global in many context

Comment (pavlovcik) [reply src/handlers/comment/handlers/first.ts:34]: Well this is in a handler specific for this feature so I think it's probably fine.

**Pull request merged**